src/components/BookingBreakdown/LineItemBookingPeriod.js

```diff
@@ -40,9 +40,12 @@ const BookingPeriod = props => {
 const LineItemBookingPeriod = props => {
   const { transaction, booking, unitType } = props;
 
-  const { start: startDate, end: endDateRaw } = booking.attributes;
-  const localStartDate = dateFromAPIToLocalNoon(startDate);
-  const localEndDateRaw = dateFromAPIToLocalNoon(endDateRaw);
+  // Attributes: displayStart and displayEnd can be used to differentiate shown time range
+  // from actual start and end. It can help in situations where there are preparation time needed
```

Comment:
Perhaps s/actual start and end/actual start and end times used for availability reservation/?